composer.json

```diff
@@ -23,8 +23,7 @@
     "require-dev": {
         "fzaninotto/faker": "~1.4",
         "phpunit/phpunit": "^8.0",
-        "mockery/mockery": "~0.9",
-        "seld/jsonlint": "^1.7"
```

Comment:
Assuming jsonlint isn't required anymore due to no flat-file dependencies?

Reply:
Correct :)